kubectl-direct-csi/util/util.go

```diff
@@ -0,0 +1,750 @@
+/*
+ * This file is part of MinIO Direct CSI
+ * Copyright (C) 2020, MinIO, Inc.
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+package util
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"regexp"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"github.com/minio/direct-csi/kubectl-direct-csi/util/randomstring"
+	"github.com/minio/direct-csi/pkg/clientset"
+)
+
+func retry(f func() error) error {
```

Comment:
we can remove this function

Reply:
Updated the method, IMO it is useful, any reason you want to remove?

Reply:
There is a retry method available already in client-go/tools/retry

Reply:
I see, let me remove this in next PR then.